### PR TITLE
added class level method to change how mathml operators are handled

### PR DESCRIPTION
**Pull request by MauriceHendrix**
## Description

Added class level method to be able to change how the transpiler handles mathml operators

## Motivation and Context

I'm currently directly messing with the internals from chaste codegen so I can control the output better, but that is a bit hacky

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] I have updated all documentation necessary.
- [ ] I have checked spelling in (new) comments.

## Testing
- [X] Testing is done automatically and codecov shows test coverage
- [ ] This cannot be tested automatically

### CODE REVIEW

File: cellmlmanip/transpiler.py [hunk 1]
```diff
@@ -58,6 +57,14 @@ def __init__(self, symbol_generator=None, number_generator=None):
         for tag_name in SIMPLE_MATHML_TO_SYMPY_NAMES:
             self.handlers[tag_name] = self._simple_operator_handler
 
+    def set_mathml_handler(mathml_operator, operator_class):
```
Comment (jonc125) [inline cellmlmanip/transpiler.py]: I think you want an `@staticmethod` decorator for this?

File: cellmlmanip/transpiler.py [hunk 2]
```diff
@@ -58,6 +57,14 @@ def __init__(self, symbol_generator=None, number_generator=None):
...
+        """Change how the transpiler handles a given mathml_operator.
+
+        :param mathml_operator: The name of a MathML operator e.g. 'exp', 'true' etc.
+        :param operator_class: A class that can handle the given operator e.g. sympy.exp, sympy.true etc.
+        """
+        SIMPLE_MATHML_TO_SYMPY_NAMES[mathml_operator] = operator_class
```
Comment (jonc125) [inline cellmlmanip/transpiler.py]: Given the values are no longer names, this dict should be renamed e.g. to `SIMPLE_MATHML_TO_SYMPY_FUNCTIONS`

Comment (jonc125) [reply cellmlmanip/transpiler.py]: Though they're not all functions either...

Comment (MauriceHendrix) [reply cellmlmanip/transpiler.py]: technically they are classes actually

**Pull request merged**